indexer/packages/postgres/__tests__/stores/affiliate-info-table.test.ts

```diff
@@ -80,7 +260,7 @@ describe('Affiliate info store', () => {
     });
 
     it('Successfully filters by address', async () => {
-      const infos: AffiliateInfoFromDatabase[] = await AffiliateInfoTable
+      const infos: AffiliateInfoFromDatabase[] | undefined = await AffiliateInfoTable
```

Comment:
why can this be undefined now?

Reply:
probably missed a merge conflict from previous PR